.github/workflows/mac.yml

```diff
@@ -0,0 +1,179 @@
+name: GraphScope CI on MacOS
+
+on:
+  # Trigger the workflow on push or pull request, but only for the main branch
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'gnn_engine/**'
+      - '**.md'
+      - '**.rst'
+      - 'docs/**'
+      - 'research/**'
+      - 'scripts/**'
+      - 'tutorials/**'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'gnn_engine/**'
+      - '**.md'
+      - '**.rst'
+      - 'docs/**'
+      - 'research/**'
+      - 'scripts/**'
+      - 'tutorials/**'
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Setup tmate session
+      if: false
+      uses: mxschmitt/action-tmate@v2
+
+    - name: Install Dependencies for MacOS
+      run: |
+        pushd ${GITHUB_WORKSPACE}
+        brew install llvm@9
+        export CC=/usr/local/opt/llvm@9/bin/clang
+        export CXX=/usr/local/opt/llvm@9/bin/clang++
+        export PATH=/usr/local/opt/llvm@9/bin:$PATH
+        export LDFLAGS="-L/usr/local/opt/llvm@9/lib"
+        export CPPFLAGS="-I/usr/local/opt/llvm@9/include"
```

Comment:
Put those repeat env settings to the job level,

see also: https://docs.github.com/en/actions/reference/environment-variables

Reply:
Don't do that for PATH, others are ok.

Reply:
why can't not do that for PATH ?

Reply:
You would lost the origin values. Ask @lidongze0629 for details.